ui/src/app/views/login.html

```diff
@@ -17,6 +17,11 @@ <h1 class="md-display-2 login-heading">Login to Bassa</h1>
     transition: 0.3s;"class="closebtn" onclick="this.parentElement.style.display='none';">&times;</span>
   Incorrect Credentials
   </div>
+  <div style="padding:14px; background-color:grey; color: white;" class="alert" ng-if="unApproved">
```

Comment:
@malithsen Moved inline styles to style sheet with [0f31c25
](https://github.com/scorelab/Bassa/pull/104/commits/0f31c257077ec03ed0dc8af9e5d67df03a655639)